client/src/components/App.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+class App extends React.Component {
+    constructor(props) {
+      super(props);
+      
+    }
+
+    render() {
+        return (
+            <div>test success!</div>
+        )
+    }
+}
+
+export default App;
```

Comment:
@LittleJennie 
@cminho921 

Two comments here:

1. I understand that this component will eventually have a lot of logic--however, at this stage, with an empty render and nothing in the constructor, this should be a stateless component.

2. If your project is following the Airbnb style guide, I'd expect to see a new line after line 16! 

Please try to be detailed in your reviews for each other to set the bar as high as possible!